x-pack/plugins/ml/public/application/model_management/test_models/models/index_input.tsx

```diff
@@ -147,6 +182,60 @@ export function useIndexInput({
   const [fieldNames, setFieldNames] = useState<Array<{ value: string; text: string }>>([]);
   const selectedField = useObservable(inferrer.getInputField$(), inferrer.getInputField());
 
+  const supportedFieldTypes = useMemo(() => inferrer.getSupportedFieldTypes(), [inferrer]);
+
+  const fetchCompanitbleIndices = useCallback(async () => {
+    try {
+      const indicesResponse = await mlApiServices.getIndices();
+
+      const indices: string[] = indicesResponse
+        .filter((index) => !index.name.startsWith('.') && !index.hidden)
+        .map((index) => index.name);
+
+      const { indices: compatibleIndices } = await mlApiServices.getFieldCaps({
+        index: indices.join(','),
+        types: supportedFieldTypes,
+        includeEmptyFields: false,
+      });
+
+      const adHocDataViews = (
+        await Promise.all(
+          (compatibleIndices as string[]).map(async (index) => {
+            const dataView = await dataViews.create({
+              title: index,
+              name: index,
+            });
+            return dataView;
+          })
+        )
+      )
+        // Fields caps response include unrelevand indcies, hence need to filter
```

Comment:
Typo - `unrelevand`. Do you mean `unsupported` or `irrelevant` ?

Reply:
fixed in https://github.com/elastic/kibana/pull/180795/commits/745fdca29d23db2075aa69239486804a83d8203c